pkg/ccl/backupccl/restore_job.go

```diff
@@ -889,15 +889,15 @@ func createImportingTables(
 			}
 		}
 	}
-	var tempSystemDBID sqlbase.ID
+	tempSystemDBID := uint32(keys.MinNonPredefinedUserDescID)
```

Comment:
Why do we need to cast to uint32?

Reply:
Done - was flipping back and forth about which side of the comparison below to cast and ended up casting both. Int will suffice.